### DISCUSSION TITLE
Problem with power-time curve simulation in Pyrosim

### DISCUSSION THREAD

**Discussion by gio309246** [firemodels/fds#13662] [Q&A]
Hello everyone,

I am a beginner with Pyrosim and I am modeling a building, but for now, I have reduced my project to a single portion of the facade (3 m wide and 10 m high, with a floor height of 3.5 m).

I have configured the facade with:
- a 30 cm reinforced concrete obstruction
- a 10 cm EPS insulation obstruction
- a void space to simulate a 10 cm air cavity
- a 5 cm Alucobond cladding obstruction
Each obstruction has been assigned the corresponding material and surface.

However, the simulations return an incorrect HRR curve. I should obtain a value of 5000 kW, but that does not happen.

I have tried to set the burner in two ways:
- 1 m x 1 m surface with an HRRPUA of 5000 kW/m².
- 2 m x 2.5 m surface (corresponding to the size of an opening, assuming the fire starts from a room on the ground floor and spreads through a patio door) with an HRRPUA of 1000 kW/m².
I have tried both, but the program does not return the HRR curve with the power of 5000 kW. Additionally, I also tried using a 10 cm mesh, but it did not yield the expected results.

I have a very fine mesh of about 5 cm, and I suspect that it might be a mesh issue. I would like to know:
1. Can the mesh size influence the simulation results?
2. What could be the causes of this discrepancy in the HRR curve?
3. How can I resolve the problem? Are there specific solutions or best practices for optimizing the mesh in this context?

Thank you in advance!

**Comment by drjfloyd** [comment]
As a beginner strongly recommend you do not start trying to immediately
model a complex geometry and scenario. Start with very simple input files
where you test the various aspects of your full problem.

**Reply by gio309246** [reply]
Thank you for your advice and response! I have already used Pyrosim for other projects and have created test models to evaluate the program's response. I noticed that with a larger mesh (10 cm, 20 cm, or more), I get the desired HRR curve. However, I need to refine the mesh. When I refine the mesh from 10 cm to 5 cm, the HRR curve no longer reaches the maximum power of 5000 kW. Why does this happen? Thank you in advance for your response.

**Reply by drjfloyd** [reply]
You have made some form of error in your input. Post a simple FDS (not Pyrosim) file that demonstrates the problem you are having.

**Reply by gio309246** [reply]
I am implementing additional test models and I believe I have found a possible solution, which involves increasing the mesh in the z direction. If there are any further developments, I will continue to post here. For now, thank you for the insights!
